11 rmarkdown

ryanpeek opened the pull request:
Ok! I think this is my final lesson (for now). Take a look...there's still a lot I didn't cover and yet this feels like a lot already. If you have additional links or good resources, feel free to add them. Thanks! 🐸

richpauloo commented:
Minor editorial changes: added an aside, image, and footnote here and there, caught a stay typo or two. 

Feel free to review and merge. The only thing I might suggest is at the very end before sharing resources is to mention that .doc, .pdf, beamer presentations, files are within the RMarkdown universe, but we don't cover them in this course due to time constraints (and on our own end, configuring latex and such). I added something to get the idea down, but feel free to change or edit.

ryanpeek commented:
Thanks! I like your suggestion, I added some additional links/resources (mentioned TinyTex specifically) and fixed a few formatting bits that I missed. I'm going to merge this in!

richpauloo reviewed (approved):
Jut looked over the changes and they look great! Way to extend it to slides. I still haven't even gone there myself, but it's good to know it exists.
